QModManager/Error.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using UnityEngine.UI;
+
+namespace QModManager
+{
+    internal static class Error
```

Comment:
Best to choose a more descriptive class name. This one could cause confusion.  
Suggestion: `internal static class ErrorDialogs`